component-runtime-beam/src/main/java/org/talend/sdk/component/runtime/beam/spi/record/AvroSchema.java

```diff
@@ -63,6 +64,8 @@ static AvroSchema toAvroSchema(final org.talend.sdk.component.api.record.Schema
 
     private List<Entry> entries;
 
+    private Map<String, Entry> entryMap;
```

Comment:
no need @JsonbTransient here?

Reply:
not totally sure as json ser/dser using object's methods but in case, will add it. 👍🏻